kytos/utils/napps.py

```diff
@@ -335,13 +336,25 @@ def create_napp(cls, meta_package=False):
         print('--------------------------------------------------------------')
         print('')
         while not cls.valid_name(username):
-            username = input('Please, insert your NApps Server username: ')
+            try:
+                username = input('Please, insert your NApps Server username: ')
+            except KeyboardInterrupt:
+                print("\nUser cancelled napps creation.")
```

Comment:
Hi @slinksoft, thanks for your contribution. I have a small suggestion, maybe you can put this message, line 342, in a variable, avoiding duplicating this message in `try except`.``